todo/src/components/ToDoList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import {addToDo} from "../actions/actions";
+
+class ToDoList extends React.Component {
+    state= {
+        newTodo: []
+    }
+    render(){
+        return(
+            <>
+            <h2>Add your Todos here!</h2>
+            <div>
+            <input
```

Comment:
I know the focus is redux, but don't forget about this still being a react app that offloads everything to the smallest component. Could your input be in another file?
Also I see handleChanges here, but I don't find the function in your code. I think this is one of the small things keeping it from working.